swagger.yaml

```diff
@@ -3447,17 +3447,52 @@ paths:
           format: uuid
           required: False
         - in: query
-          name: host_id
+          name: hosts_id
```

Comment:
isn't it an API change?
regardless of UI being adjustable, there might be other consumers of this endpoint which might be affected

Reply:
By only adding the parameter ``hosts_id`` to the existing parameters, documenting that it takes precedence over ``host_id``
``host_id`` can be documented to be deprecated, and also annotated ``deprecated: true`` (https://swagger.io/docs/specification/describing-parameters/)

Reply:
I believe it can be annotated only in `OA 3.0 API` I can add (DEPRECATED) in the description

Reply:
no problem. just search the swagger for similar cases where we've changed parameters to their plural variant